### DISCUSSION TITLE
Not work with Objective-C mixed Swift project

### DISCUSSION THREAD

**Discussion by liangwei518** [ccache/ccache#265] [Support]
My project uses Objective-C and Swift intermixing. When I use ccache, the statistics are correct, but the compile time is increased.

  ~ ccache -s
cache directory                     /Users/summer/.ccache
primary config                      /Users/summer/.ccache/ccache.conf
secondary config      (readonly)    /usr/local/Cellar/ccache/3.4.2/etc/ccache.conf
stats zero time                     Tue May 22 10:52:44 2018
cache hit (direct)                 17242
cache hit (preprocessed)            6576
cache miss                          4779
cache hit rate                     83.29 %
cleanups performed                     0
files in cache                     18019
cache size                         562.7 MB
max cache size                       5.0 GB

What would cause such a situation?
Looking forward to your answer.

**Comment by jrosdahl** [comment]
It's expected that the compile time is increased the first time, but the second should be faster. See also http://ccache.samba.org/performance.html. If this is not what you see you need to explain your scenario in more detail.
